tensorflow/contrib/eager/python/examples/notebooks/eager_intro.ipynb

```diff
@@ -1,27 +1,107 @@
 {
```

Comment:
This seems to be under the wrong name still.  eager_intro --> eager_basics right?

Reply:
fixed